aws/centralized-monitoring/variables.tf

```
@@ -0,0 +1,9 @@
+variable "ternary_service_account_email" {
+  type        = string
+  description = "Service Account email used by your Ternary tenant to access your cloud resources. Available in your Admin page."
+}
+
+variable "ternary_service_account_uid" {
+  type        = string
+  description = "Service Account Unique ID used by your Ternary tenant to access your cloud resources. Available in your Admin page."
+}
```

What do you think about adding a [variable validation](https://developer.hashicorp.com/terraform/language/values/variables#custom-validation-rules) to ensure the variables conform to the expected values?

Maybe something like this
```suggestion
variable "ternary_service_account_uid" {
  type        = string
  description = "Service Account Unique ID used by your Ternary tenant to access your cloud resources. Available in your Admin page."
  validation {
    condition     = length(regexall("[0-9]+", var.ternary_service_account_uid)) > 0
    error_message = "The ternary_service_account_uid value must be numeric."
  }
}
```